Cookbook.API/ApiEndPoints.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Cookbook.API
+{
+    public static class ApiEndPoints
```

Comment:
En amugy ilyen proper REST api-n sose dolgoztam cegnel, szoval ha valamivel ellent mondok a YouTube-os arcoknak, akkor lehet nekik van igazuk.

Ez pl egy pattern amit a videoban lattal, vagy te akartad szetvalasztani? Szerintem ez ront a dolgon inkabb mint segit. En nem akarnek kulon file-ba menni hogy lassam hogy mi az ertek.

Annak lehet ertelme hogy a Base-t kirakod egy const-ba a file-on belul, es akkor `$"{Base}/{{idOrSlug}}"`-t irsz a `[]`-be, de nem raknam ki oket kulon file-ba. Nem hasznalod egyiket tobbszor amugy vagy mas file-okbol, szoval en nem latok sok erteket egy kulon class-ban.

Reply:
Videoban volt. Talan az ilyen hosszabbaknal szebb "titles/tags/{tag}" de osszesegeben tell nem segit sokat. a Baset ki lehet rakni az elejere
```
namespace Cookbook.API.Controllers
{
    [ApiController]
    [Route("api/[controller]")]
    public class RecipesController : ControllerBase
```